spt/OrangeBox/patterns.hpp

```diff
@@ -318,17 +334,49 @@ namespace Patterns
 		{
 			"2257546",
 			{
-				0x55, 0x8B, 0xEC, 0x83, 0xEC, 0x14, 0x56, 0x8B, 0xF1, 0x8B, 0x4E, 0x04, 0x80, 0xB9, 0x2C, 0x10, 0x00, 0x00, 0x00, 0x74, 0x0E, 0x8B, 0x46, 0x08, 0x83, 0x48, 0x28, 0x02, 0x32, 0xC0, 0x5E, 0x8B, 0xE5, 0x5D, 0xC3, 0xF3, 0x0F, 0x10, 0x89, 0x28, 0x11, 0x00, 0x00
+				0x55, 0x8B, 0xEC, 0x83, 0xEC, 0x14, 0x56, 0x8B, 0xF1, 0x8B, 0x4E, 0x04, 0x80, 0xB9, '?', '?', '?', '?', '?', 0x74, 0x0E, 0x8B, 0x46, 0x08, 0x83, 0x48, 0x28, 0x02, 0x32, 0xC0, 0x5E, 0x8B, 0xE5
 			},
-			"xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx"
+			"xxxxxxxxxxxxxx?????xxxxxxxxxxxxxx"
 		},
 
 		{
-			"2257546-HL1",
+			"2257546-hl1",
 			{
-				0x55, 0x8B, 0xEC, 0x83, 0xEC, 0x08, 0x56, 0x8B, 0xF1, 0x8B, 0x4E, 0x04, 0x80, 0xB9, 0xB4, 0x0F, 0x00, 0x00, 0x00, 0x74, 0x0E, 0x8B, 0x46, 0x08, 0x83, 0x48, 0x28, 0x02, 0x32, 0xC0, 0x5E, 0x8B, 0xE5, 0x5D, 0xC3, 0xF3, 0x0F, 0x10, 0x81, 0xB0, 0x10, 0x00, 0x00
+				0x55, 0x8B, 0xEC, 0x51, 0x56, 0x8B, 0xF1, 0x57, 0x8B, 0x7E, 0x04, 0x85, 0xFF, 0x74, 0x2B, 0x8B, 0x07, 0x8B, 0xCF, 0x8B, 0x80,  '?', '?', '?', '?', 0xFF, 0xD0, 0x84, 0xC0, 0x74, 0x1B, 0x6A, 0x00
 			},
-			"xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx"
+			"xxxxxxxxxxxxxxxxxxxxx????xxxxxxxx"
+		},
+
+		{
+			"2707",
+			{
+				0x83, 0xEC, 0x10, 0x53, 0x56, 0x8B, 0xF1, 0x8B, 0x4E, 0x08, 0x8A, 0x81, '?', '?', '?', '?', 0x84, 0xC0, 0x74, 0x0F, 0x8B, 0x76, 0x04, 0x83, 0x4E, 0x28, 0x02, 0x5E, 0x32, 0xC0, 0x5B, 0x83, 0xC4, 0x10
+			},
+			"xxxxxxxxxxxx????xxxxxxxxxxxxxxxxxx"
+		},
+
+		{
+			"4044-episodic",
+			{
+				0x83, 0xEC, 0x10, 0x56, 0x8B, 0xF1, 0x8B, 0x4E, 0x08, 0x80, 0xB9, '?', '?', '?', '?', '?', 0x0F, 0x85, '?', '?', '?', '?', 0xD9, 0x05, '?', '?', '?', '?', 0xD9, 0x81, '?', '?', '?', '?'
+			},
+			"xxxxxxxxxxx?????xx????xx????xx????"
```

Comment:
Don't leave wildcard bytes in the end of patters, they do nothing but slow down the search unnecessarily